toml/longrun_1M.toml

```diff
@@ -0,0 +1,15 @@
+[zd_rayleigh]
+value = 35000.0
+type = "float"
+
+[alpha_rayleigh_uh]
+value = 0.0
+type = "float"
```

Comment:
```suggestion
```

If I understand this correctly the default value has been updated here, we shouldn't need this block : https://github.com/CliMA/CLIMAParameters.jl/blob/110de02b2c1d12673b130ae94295e1628bdf7f4e/src/parameters.toml#L1745